src/systems/filecoin_mining/storage_mining/storage_miner_actor_state.go

```diff
@@ -65,18 +103,18 @@ func (st *StorageMinerActorState_I) _updateSectorUtilization(rt Runtime, lastPoS
 
 }
 
-func (st *StorageMinerActorState_I) _getActivePower(rt Runtime) block.StoragePower {
+func (st *StorageMinerActorState_I) _getActivePower() (block.StoragePower, error) {
 	var activePower = block.StoragePower(0)
 
 	for _, sectorNo := range st.SectorTable().Impl().ActiveSectors_.SectorsOn() {
 		utilizationInfo, found := st.SectorUtilization()[sectorNo]
```

Comment:
please use `_getUtilizationInfo` here which does this safely and is used everywhere else

Reply:
But this is safe, instead of dealing with error internally though I return it so it can be handled by the caller appropriately (rt error for actors, something else for subsystems). Can you explain why `_getUtitlizationInfo` is more appropriate?